go-app-ussd_registration.js

```diff
@@ -578,11 +580,24 @@ go.app = (function() {
       return new FreeText(name, {
         question: get_content(name).context(),
         check: function(content) {
+          var givendate = content;
+          givendate = new Date(givendate);
+          var today = new Date();
+          today.setHours(0,0,0,0);
+          var timestamp = new Date().getTime() + (400 * 24 * 60 * 60 * 1000);
```

Comment:
I'm not sure what we are doing here? can we add a comment

Reply:
In the rapidpro flow we check if the next clinic visit date entered is more than 400 days from today. If true we return that error. So I was just setting the date I used to compare 400 days from today.